### PR TITLE
BPartner org change mass

### PR DESCRIPTION
**Pull request by metas-rc**

### CODE REVIEW

File: backend/de.metas.contracts/src/main/java/de/metas/contracts/bpartner/process/C_BPartner_MoveToAnotherOrg_Mass.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+import javax.annotation.Nullable;
+import java.time.Instant;
+import java.util.Iterator;
+
+public class C_BPartner_MoveToAnotherOrg_Mass extends JavaProcess implements
+		IProcessDefaultParametersProvider
+{
+
+	public static final String PARAM_AD_ORG_SOURCE_ID = "AD_Org_From_ID";
```
Comment (cristinamghita) [inline backend/de.metas.contracts/src/main/java/de/metas/contracts/bpartner/process/C_BPartner_MoveToAnotherOrg_Mass.java:51]: This parameter is not used. Please, clean it.

File: backend/de.metas.contracts/src/main/java/de/metas/contracts/bpartner/process/C_BPartner_MoveToAnotherOrg_Mass.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+	@Param(parameterName = PARAM_IsCloseInvoiceCandidate, mandatory = true)
+	private boolean isCloseInvoiceCandidate;
+	@Param(parameterName = PARAM_WhereClause, mandatory = true)
+	protected String p_WhereClause;
+
+	@Override
+	protected String doIt() throws Exception
+	{
+		final Iterator<I_C_BPartner> partnerIterator = retrievePartnersToMove(p_WhereClause);
+
```
Comment (cristinamghita) [inline backend/de.metas.contracts/src/main/java/de/metas/contracts/bpartner/process/C_BPartner_MoveToAnotherOrg_Mass.java:75]: Please, use ```de.metas.bpartner.service.IBPartnerDAO#retrieveBPartnerIdsBy``` , if possible.
if not, please move the method in  `IBPartnerDAO`

File: backend/de.metas.contracts/src/main/java/de/metas/contracts/bpartner/process/C_BPartner_MoveToAnotherOrg_Mass.java [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+		while (partnerIterator.hasNext())
+		{
+			final I_C_BPartner partner = partnerIterator.next();
+
+			final OrgChangeRequest orgChangeRequest = OrgChangeRequest.builder()
+					.bpartnerId(BPartnerId.ofRepoId(partner.getC_BPartner_ID()))
```
Comment (cristinamghita) [inline backend/de.metas.contracts/src/main/java/de/metas/contracts/bpartner/process/C_BPartner_MoveToAnotherOrg_Mass.java:81]: I would have expected here to use AD_Org_From_ID and this param to be mandatory.
And using `p_WhereClause`, to filter the parameters from the given org,

**Review by cristinamghita — COMMENTED** [review]
Check my comments

File: backend/de.metas.contracts/src/main/java/de/metas/contracts/bpartner/process/C_BPartner_MoveToAnotherOrg_Mass.java (hunk 3 above)
Comment (metas-rc) [reply backend/de.metas.contracts/src/main/java/de/metas/contracts/bpartner/process/C_BPartner_MoveToAnotherOrg_Mass.java:82]: The issue giver (Torby) specifically asked to not have the parameter and to rely only on the whereclause for the partner selection.

**Review by cristinamghita — APPROVED** [review]
lgtm

**Pull request merged**